apps/ecommerce/frontend/src/components/App/Dialog/ResourceCard/ResourceCard.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import ResourceCard from './ResourceCard';
+import { externalResource } from '../../../../../test/mocks';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+
+const { getByText, getByTestId } = screen;
+
+describe('ResourceCard component', () => {
+  it('mounts', () => {
+    render(
+      <ResourceCard
+        cardHeader="Kleenex product"
+        onSelect={() => {}}
+        resource={externalResource}
+        selectedResources={[]}
+      />
+    );
+
+    const productName = externalResource.name!;
+    const productDescription = externalResource.description!;
+
+    expect(getByText(productName)).toBeVisible();
+    expect(getByText(productDescription)).toBeVisible();
+  });
+
+  it('handles onSelect', async () => {
+    userEvent.setup();
+    const mockOnSelect = jest.fn();
+    render(
+      <ResourceCard
+        cardHeader="Kleenex product"
+        onSelect={mockOnSelect}
+        resource={externalResource}
+        selectedResources={[]}
+      />
+    );
+
+    const productName = externalResource.name!;
+    await userEvent.click(getByText(productName));
+
+    const card = getByTestId('cf-ui-card');
+
+    expect(mockOnSelect).toHaveBeenCalled();
+    expect(card).toBeEnabled();
```

Comment:
Is this the right assertion here? I am wondering if this will return true, regardless of whether the card has been selected?

Reply:
Yeah good point! It is always enabled, looks like the card itself doesn't have an easy attribute to assert being active vs not active. I have removed for now, especially with the future changes of this card 👍